test/check-application

```diff
@@ -885,13 +885,11 @@ class TestApplication(testlib.MachineCase):
     @testlib.nondestructive
     def testCheckpointRestoreCGroupsV2(self):
         b = self.browser
-        m = self.machine
 
         if not self.has_cgroupsV2:
             return
 
-        # Only Fedora works with CGroupsV2
-        if m.image.startswith('fedora'):
+        if self.has_criu:
```

Comment:
@marusak: I'm not 100% sure about this change -- conceptually, hardcoding "fedora" here seems wrong and error prone. In practice, RHEL 9 has cgroups v2 and a recent criu (same as F34), but throws "does not support checkpoint/restore". But I'd consider that a bug (filing now), not a behavior to validate. It also does not help with `testCheckpointRestoreCGroupsV1` where rhel-9 fails in the same way, and this hack doesn't help us.

Reply:
I filed https://bugzilla.redhat.com/show_bug.cgi?id=1944964 and I'm about to do a naughty for it -- let me know if you think that's the right course of action, or you'd rather hack the tests.

Reply:
If they don't want to support criu in RHEL 9, then they should drop the `Recommends: criu`, and our image preparation can anticipate that with `rpm -e criu` and setting `has_criu` to False on RHEL-9.